atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepDelay.java

```diff
@@ -41,11 +42,13 @@
 
     private final long initialPause;
     private final long maxPauseMillis;
+    private final LongConsumer updateMetrics;
```

Comment:
nit: this name is a bit meh, I would normally name this like `sweepDelayMetricsUpdater` or something. Not too concerned as it's in the SweepDelay class.